src/environmentd/src/http.rs

```diff
@@ -339,6 +347,12 @@ impl InternalServer {
             .route(
                 "/api/livez",
                 routing::get(mz_http_util::handle_liveness_check),
+            )
+            .route(
+                "/enable_otel",
```

Comment:
To make this more REST-y let's do a PUT to /api/opentelemetry/config:

```suggestion
                "/api/opentelemetry/config",
```

That'll give us the flexibility to add additional configuration parameters down the road, too.